### ISSUE TITLE
[iceandfire-2.1.0-1.16.3]  Sea Serpent,  Dragon Skeletons, Hippocampus not spawn

### ISSUE THREAD

**Issue by Daejangnamu** [Alex-the-666/Ice_and_Fire#2977]
#1 out of this number chance per chunk for generation
#Range: 1 ~ 10000
"Generate Dragon Skeleton Chance" = 1

#1 out of this number chance per chunk for generation
#Range: 1 ~ 10000
"Spawn Sea Serpent Chance" = 1

#1 out of this number chance per chunk for generation
	#Range: 1 ~ 10000
	"Spawn Hippocampus Chance" = 1

#biome list
seaSerpentBiomes = ["minecraft:ocean", "minecraft:deep_ocean", "minecraft:frozen_ocean", "minecraft:warm_ocean", "minecraft:lukewarm_ocean", "minecraft:cold_ocean", "minecraft:deep_warm_ocean", "minecraft:deep_lukewarm_ocean", "minecraft:deep_cold_ocean", "minecraft:deep_frozen_ocean"]

#biome list
fireDragonSkeletonBiomes = ["minecraft:desert", "minecraft:desert_hills", "minecraft:desert_lakes", "biomesoplenty:cold_desert", "biomesoplenty:dryland", "biomesoplenty:lush_desert", "biomesoplenty:outback", "biomesoplenty:wasteland"]

My new overworld not spawn
Please check

**Comment by CommandBlockGuru** [comment]
Maybe try turning up the chances a bit?

**Comment by Zhaile** [comment]
Just confirming that I am having this issue in 1.16.3. I have cranked the Hippocampus spawn chance up to 10000 just to see if any would spawn. I was >4000 blocks from spawn, over deep ocean. None spawned even after moving to new chunks with the max spawn setting.

Also, I have not seen any trolls spawn. To look for them, I went out several thousand blocks, gave myself night vision, stayed in the chunks to load mobs, and then went underground as a spectator. I saw all vanilla mobs and dragons caves (with dragons). I noticed that the only mobs spawning in the couple of dragon caves that I looked at were bats and creepers. Is there some new logic trying to restrict what mobs spawn in lairs? Or maybe this was just a fluke.

Edit: I wanted to mention that I removed all other addons aside from I&F and Citadel for the above testing.
Edit 2: I did have datapacks installed. I removed those, loaded up a world, and teleported far out over an ocean using the seed. Still zero Hippocampi (still max spawn chance) and zero sea serpents. 

Versions: Minecraft 1.16.3, I&F - 2.0.3, Citadel 1.4.0, Forge 34.1.0
